### ISSUE TITLE
Composer installation with `--stability=dev` of TYPO3 fails

### ISSUE THREAD

**Issue by DavidBruchmann** [IchHabRecht/multicolumn#58]
also with `--prefer-stable`.
Complaint is that it can't be resolved between 7.x-dev and 8.x-dev.

**Comment by IchHabRecht** [comment]
Hi @DavidBruchmann,

Would you mind to share some more information about what you are trying to do? Multicolumn needs an official released TYPO3 as dependency. If you want to use any develop branch, you need to use "7.x-dev as 7.6.99"

**Comment by DavidBruchmann** [comment]
Hi Nicole,

I never had a special purpose installing the dev-branch, I thought being able to circumvent some problems with extensions perhaps when I decided to install the dev-branch.
I remarked that my decision was wrong as I succeeded after switching to stable.
I have to mention that my experience with composer is limited.
So I only wanted to report the issue, but if it's working with your recipe that's ok for me.
Thank's for the unexpected quick answer ;-)
I think the issue can be closed then.
